cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -153,9 +159,9 @@ image::rosa-create-cli-16.png[]
 +
 image::rosa-create-cli-billing-17.png[]
 +
-* Only AWS accounts that were linked to the Red{nbsp}Hat organization that is currently used will be shown.
-* The specified AWS account will be charged for using the ROSA service, regardless of whether the infrastructure AWS account is linked to it in the same AWS organization.
-* You can see an indicator of whether the ROSA contract is enabled for a given AWS billing account or not.
+* Only AWS accounts that were linked to the Red{nbsp}Hat organization that is currently used are shown.
+* The specified AWS billing account is charged the ROSA service fee. Cloud resources, such as the EC2 instance costs, are billed to the AWS infrastructure account where the resources are deployed. Depending on your AWS organization structure, different AWS organization accounts might be responsible for the ROSA service fee and for the AWS infrastructure fees.
+* AWS accounts without the _Contract enabled_ label are charged the applicable on-demand consumption rates.
 * To select an AWS account that does not have the contract enabled, refer to the first few steps in this tutorial to enable the contract and allow the service charging, which is required for a successful cluster deployment.
```

Comment:
This bullet point is incorrect and needs to be completely removed (users can select billing accounts with or without the "contract enabled" label, the difference is only that the ones with "contract enabled" consume the previously purchased quota first, before being charged the on-demand service fees).